package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-html-element",
-  "version": "0.0.1",
+  "version": "1.0.0",
```

Comment:
Just curious was this supposed to have been bumped up earlier? Asking because this doesn't seem like a major change.

Reply:
Should have been, but this is harmless since it's just the lock file.

Reply:
Like Alex said, this is getting updated because the package.json is already at v1, but the package-lock hadn't been regenerated with the new version.

Ultimately, it shouldn't matter since `semantic-release` should update it anyway.